src/components/dialog/GitNewBranchDialog.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <default-dialog dialog-key="GitNewBranch" data-cy="git-newBranch-dialog">
+    <template v-slot:title>
+      <q-icon color="primary" name="fa-brands fa-git-alt" />
+      {{ $t('page.modelizer.git.newBranch.title') }}
+    </template>
+    <template v-slot:default>
+      <git-new-branch-form
+          :project-name="projectName"
+          :branch-name="branchName"
+          @git-branch:create="DialogEvent.next({ type: 'close', key: 'GitNewBranch' })"/>
```

Comment:
5 indents instead of 4 and put the closing tag on next line

Reply:
2 indent *

Reply:
done